change/@microsoft-fast-components-2fd58c1a-a158-410b-8690-fe64e28f8440.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix broken disclosure",
```

Comment:
can we update this to be a bit more descriptive of what exactly is being solved?

Reply:
ok